contrib/src/main/java/gov/nasa/jpl/aerie/contrib/streamline/debugging/Logger.java

```diff
@@ -0,0 +1,60 @@
+package gov.nasa.jpl.aerie.contrib.streamline.debugging;
+
+import gov.nasa.jpl.aerie.contrib.streamline.core.Resources;
+import gov.nasa.jpl.aerie.merlin.framework.Registrar;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Map;
+
+import static gov.nasa.jpl.aerie.merlin.protocol.types.Duration.MICROSECONDS;
+import static java.util.stream.Collectors.toMap;
+
+public class Logger {
+    private static final int LEVEL_INDICATOR_SIZE = Arrays.stream(LogLevel.values())
+            .map(v -> v.toString().length())
+            .max(Integer::compareTo)
+            .orElseThrow();
+    private static final String LOG_MESSAGE_FORMAT = "%s [%-" + LEVEL_INDICATOR_SIZE + "s] %s";
+
+    private final Map<LogLevel, SimpleLogger> subLoggers;
+    private final Instant planStart;
+
+    public Logger(Registrar registrar, Instant planStart) {
+        // TODO - refactor the plan start out, by instead building a global sim clock to do the same job.
```

Comment:
@DavidLegg and I discussed that this TODO comment should be addressed before merging this PR. In particular - there's no need to explicitly include the log timestamp in the message, because every serializable event is time-tagged